src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setError] = useState(false);
+  const [isLoaded, setLoaded] = useState(false);
+
+  useEffect(() => {
+    setLoaded(false);
```

Comment:
This is not needed considering, you have `[]` as a useEffect dependency list. It will be once in a components lifecycle anyway
```suggestion
```

Reply:
Fixed

Reply:
Fixed